### PR TITLE
Fix beam_PerformanceTests_Python

### PR DESCRIPTION
**Pull request by markflyhigh**
This PR continue the fix work from https://github.com/apache/beam/pull/6074.

Fix Python wordcount benchmark running on Jenkins branch `beam_PerformanceTests_Python`.
- Build Python tar ball to target directory that's required for pipeline execution.
- Only run `WordCountIT.test_wordcount_it` for this benchmark.
- Create a bigquery table `beam_performance.wordcount_py_pkb_results` with correct schema for this benchmark to store perf metrics.
- Fix setup.py so that we can use this file to install python dependencies from non-root directory.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by markflyhigh** [comment]
Run Seed Job

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
Run Seed Job

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
Run Seed Job

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
Run Seed Job

**Comment by markflyhigh** [comment]
Run Python Performance Test

**Comment by markflyhigh** [comment]
@aaltay It's ready to review. Please take a look. Thanks!

cc: @DariuszAniszewski

**Comment by DariuszAniszewski** [comment]
@markflyhigh Thanks for fixing it. 

The changes LGTM but I don't see how this fixes the stucked Dataflow job and how this connects to the commit I found in #6074 that seems to started job to fail. Was it just a coincidence? Can you please shed some light on this?

**Comment by markflyhigh** [comment]
Run Seed Job

**Comment by markflyhigh** [comment]
Run Python Performance Test

File: .test-infra/jenkins/CommonJobProperties.groovy [hunk 1]
```diff
@@ -309,31 +309,40 @@ class CommonJobProperties {
...
         // Clone appropriate perfkit branch
-        shell('git clone https://github.com/GoogleCloudPlatform/PerfKitBenchmarker.git')
+        shell("git clone https://github.com/GoogleCloudPlatform/PerfKitBenchmarker.git ${perfkit_root}")
 
         // Install job requirements for Python SDK.
-        shell('.beam_env/bin/pip install -e ' + CommonJobProperties.checkoutDir + '/sdks/python/[gcp,test]')
+        shell("${beam_env}/bin/pip install -e ${beam_root}/sdks/python/[gcp,test]")
+
+        // Build PythonSDK tar ball.
+        shell("(cd ${beam_root}/sdks/python && ${beam_env}/bin/python setup.py sdist --dist-dir=target)")
```
[the remark this answers is not in the document]
Comment (DariuszAniszewski) [reply .test-infra/jenkins/CommonJobProperties.groovy]: @markflyhigh thanks for clarifications 👍

**Pull request merged**